api/app_analytics/influxdb_wrapper.py

```diff
@@ -56,7 +56,8 @@ def write(self):
     def influx_query_manager(
         date_range: str = "30d",
         date_stop: str = "now()",
-        drop_columns: str = "'organisation', 'organisation_id', 'type', 'project', 'project_id'",
+        drop_columns: str = "'organisation', 'organisation_id', 'type', 'project', 'project_id', 'environment', \
```

Comment:
This should be pulled out as a constant.

Reply:
OK sure can you do that!